human_general_analysis/nbconverted/Viz_simulated_experiments.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Visualize simulated experiment
+# 
+# This notebook will make volcano plots of 3 representative simulated experiments to demonstrate what the simulation approach creates from a biological perspective (i.e. experiments created contain a similar experiment structure but different biological patterns)
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import pandas as pd
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt
+
+from ponyo import utils
+
+
+# In[2]:
+
+
+# Read in config variables
+base_dir = os.path.abspath(os.path.join(os.getcwd(), "../"))
+
+config_filename = os.path.abspath(
+    os.path.join(base_dir, "configs", "config_human_general.tsv")
+)
+
+params = utils.read_config(config_filename)
+
+
+# In[3]:
+
+
+# Load params
+local_dir = params["local_dir"]
+project_id = params['project_id']
+
+simulated_DE_stats_dir = os.path.join(
+    local_dir, 
+    "DE_stats"
+)
+template_DE_stats_filename = os.path.join(
+    local_dir, 
+    "DE_stats",
+    f"DE_stats_template_data_{project_id}_real.txt"
+    
+)
+
+
+# In[4]:
+
+
+# Plot volcano plots
+def make_volcano_plot_template(
+    template_DE_stats_filename,
+    project_id,
+    out_filename
+):
+    
+    # Read template DE stats
+    template_DE_stats_df = pd.read_csv(
+        template_DE_stats_filename,
+        sep="\t",
+        index_col=0,
+        header=0
+    )
+    
+    # Take -log10 of adjusted p-value
+    template_DE_stats_df["padj_log10"] = -np.log10(template_DE_stats_df["padj"])
+
+    # Label DEGs by traditional criteria
+    # log2FC > 1
+    # padj < 0.05
+    template_DE_stats_df["gene group"] = "none"
+    template_DE_stats_df.loc[(abs(template_DE_stats_df["log2FoldChange"])>1) &
+                          (template_DE_stats_df["padj"] <0.05),
+                              "gene group"
+                         ] = "DEG"
+
+    # Plot
+    colors = ["lightgrey", "#2c7fb8"]
+
+    f = sns.scatterplot(
+       data=template_DE_stats_df,
+        x="log2FoldChange",
+        y="padj_log10",
+        hue="gene group",
+        hue_order=["none", "DEG"],
+        style="gene group",
+        markers={
+            "none": ".",
+            "DEG": "o",
+        },
+        palette=colors,
+        linewidth=0,
+        alpha=0.5,
+        )
+    
+    f.set_xlabel("log2 Fold Change", fontsize=14, fontname="Verdana")
+    f.set_ylabel("-log10(FDR adjusted p-value)", fontsize=14, fontname="Verdana")
```

Comment:
```suggestion
    f.set_ylabel(
      "-log10(FDR adjusted p-value)", 
      fontsize=14, 
      fontname="Verdana"
     )
```
Add white space for above and the next line below